migration-roadmap/README.md

```diff
@@ -0,0 +1,174 @@
+# Spectrum 2 migration roadmap
+
+This directory contains comprehensive migration documentation for 2nd generation Spectrum Web Components based on the implementation of Spectrum 2 components that were previously migrated in [Spectrum CSS](https://github.com/adobe/spectrum-css/tree/spectrum-two). It helps engineers understand what needs to be implemented, updated, or aligned between the two systems to guide development of 2nd generation web components.
+
+## Purpose
+
+### Why we do this analysis
+
+The migration roadmap facilitates development of 2nd generation, full-fidelity Spectrum 2 web components by providing:
+
+- **Feature gap analysis**: Identifies new features and capabilities needed to match Spectrum 2 design specifications
+- **Implementation comparison**: Maps CSS selectors to existing SWC properties and notes markup differences between systems
+- **Deprecation tracking**: Documents features or variants being removed in Spectrum 2
+- **Planning guidance**: High-level overview of changes to assist with estimating migration effort
+
+### The bigger picture
+
+When engineers begin 2nd generation Spectrum 2 implementation work, this documentation provides:
+
+- **Clear starting point**: Entry point for development work that identifies specific areas needing adjustment
+- **Gap analysis**: Differences that would prevent Spectrum 2 CSS from working in SWC
+- **Reference materials**: Links to CSS and React implementations, and previous migration work
+
+## What we analyze
+
+### Component specifications
+
+We extract and document:
+
+- **CSS selectors**: All selectors from `metadata.json` in the spectrum-css `spectrum-two` branch
+- **Passthroughs**: CSS passthrough properties
+- **Modifiers**: CSS modifier classes
```

Comment:
Might be good to also flag that these are deprecated in S2 (so this information is mostly for migration guide purposes).

Reply:
Good reminder! Updated this and added it to the `<summary>` in the component markdown files I did here.